docs/user-guide/python.md

```diff
@@ -290,4 +290,65 @@ Please follow these steps.
 
 !!! note
     You willl of course need to use a different hostname for the SSH tunnel if you're running on the
-    ARCHER2 4-cabinet system, `login-4c.archer2.ac.uk`.   
\ No newline at end of file
+    ARCHER2 4-cabinet system, `login-4c.archer2.ac.uk`.   
+
+## Using Dask Job-Queue on ARCHER2
+
+The Dask-jobqueue project makes it easy to deploy Dask on ARCHER2. 
+You can find more information [here](http://jobqueue.dask.org/en/latest/).
```

Comment:
Can you change "here" to "in the Dask Job-Queue documentation". The use of "here" for link text is problematic for accessibility reasons.